config/initializers/filter_parameter_logging.rb

```diff
@@ -2,4 +2,4 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [:password, :suse_registry_mirror_certificate]
```

Comment:
I think we want here `suse_registry_mirror_cert`

Reply:
Yep.